modal/dict.py

```diff
@@ -521,8 +527,8 @@ async def put(self, key: Any, value: Any, *, skip_if_exists: bool = False) -> bo
         try:
             resp = await self._client.stub.DictUpdate(req)
             return resp.created
-        except GRPCError as exc:
-            if "status = '413'" in exc.message:
+        except ResourceExhaustedError as exc:
+            if "status = '413'" in str(exc):
```

Comment:
We do this check so often that it could be it's own function. (With a more descriptive function name)